mautic/mautic
# DPMMA-1444 Emails bar chart 7.x

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🟢 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

Adding a bar chart to the raport with e-mail data: sent, read, bounced, unsubscribed.

![image](https://github.com/user-attachments/assets/6d1c153e-da3f-4c03-bf19-255c05792493)









---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create and send several emails
3. Open and Read sent emails
4. Create a report with data source emails, and Graph Read / Ignored / Unsubscribed / Bounced (Bar chart)
5. Check the results
